Makefile

```diff
@@ -336,6 +342,10 @@ lint-code: $(GOLANGCI_LINT)
 	@printf $(COLOR) "Linting code..."
 	@$(GOLANGCI_LINT) run --verbose --timeout 10m --fix=$(GOLANGCI_LINT_FIX) --new-from-rev=$(GOLANGCI_LINT_BASE_REV) --config=.golangci.yml
 
+lint-imports: $(GCI_LINT) # Don't get confused, there is a single linter called gci, and the mega linter we use is called golangci-lint.
+	@printf $(COLOR) "Linting imports..."
+	@$(GCI_LINT) write --skip-generated -s standard -s default ./*
```

Comment:
Should the target maybe be `fmt-imports`? Since it's using `write`, will it ever fail?

Reply:
That also removes the need for the clarifying comments about the linters.

Reply:
No, I don't think so. I think its best to follow the naming convention of the other targets called by the linters workflow.

Relatedly, I forgot that there are goimport targets. I'm going to add this as a dependency to those and also merge goimports and goimports-all since there shouldn't be any unsorted statements allowed into main anymore.

Reply:
Okay, 2 is enough for me.